messages/teleporter/message_handler.go

```diff
@@ -183,47 +180,36 @@ func (m *messageHandler) ShouldSendMessage(destinationClient vms.DestinationClie
 		return false, nil
 	}
 
```

Comment:
```suggestion
// Dispatch to the external decider service. If the service is unavailable or returns
// an error, then use the decision that has already been made, i.e. return true
```

Let's add a comment to better separate the two "halves" of this function.